src/js/views/patients/patient/dashboard/layout.hbs

```diff
@@ -8,7 +8,7 @@
 </div>
 <div class="patient__actions" data-add-workflow-region>
   <span class="button-primary">
-    <span class="js-loading">{{far "circle-plus"}}{{ @intl.patients.patient.dashboard.dashboardViews.actionLoading }}</span>
```

Comment:
actually I think only 
```
<span class="js-loading patient__actions-loading">
  {{far "circle-plus"}}
  <span>{{ @intl.patients.patient.dashboard.dashboardViews.actionLoading }}</span>
</span>
```
fixes this.  No need for other styles.

Reply:
Nice, much better way.